src/components/HeaderNavMenu.jsx

```diff
@@ -30,8 +52,27 @@ function HeaderNavMenu() {
           </MuiLink>
         </Grid>
       ))}
-      <Grid item>
-        <LoginButton className={classes.menuItem} />
+      <Grid item className={classes.avatarMenu}>
+        {(loading && <null />) ||
+          (isSignedIn ? (
+            <ProfileMenu
+              signOut={signUserOut}
+              name={user.displayName}
+              email={user.email}
+              photo={user.photoURL}
+            />
+          ) : (
+            <Button
+              variant="contained"
+              color="primary"
+              component={Link}
+              to={{
+                state: { showLoginDialog: true },
+              }}
```

Comment:
This could be written as 
```suggestion
              to={{state: { showLoginDialog: true }}}
```

Reply:
Ok I changed it